consensus/core/src/authority_service.rs

```diff
@@ -104,11 +108,12 @@ impl<C: CoreThreadDispatcher> NetworkService for AuthorityService<C> {
 
         // Reject blocks failing validations.
         if let Err(e) = self.block_verifier.verify(&signed_block) {
+            let error: &'static str = e.clone().into();
```

Comment:
That will derive the error enum name only, not parameters etc, so cardinality will be low.

Reply:
Maybe add a test for this in BlockVerifier?